### ISSUE TITLE
aws.StaticSite errorPage issue

### ISSUE THREAD

**Issue by iOSonntag** [sst/ion#481]
Version: 0.0.393

Currently it is not possible to use the errorPage field in the StaticSite construct.

```ts
const website = new sst.aws.StaticSite('Website', {
      path: 'website',
      indexPage: 'index.html',
      // wont work 
      // errorPage: 'index.html',
      // errorPage: '/index.html',
      domain: {
        name: 'example.com',
        redirects: [
          'www.example.com',
        ],
      },
      transform: {
        cdn: {
          customErrorResponses: [
            // works
            {
              errorCode: 404,
              responsePagePath: '/index.html',
              responseCode: 200,
            },
          ],
        }
      }
});
```
The deploy fails with message:
`CloudFront: UpdateDistribution, https response error StatusCode: 400, RequestID: XXX, IllegalUpdate: Your request must specify both ResponsePagePath and ResponseCode together or both should be empty.`

**Comment by mlafeldt** [comment]
I'm also getting the same CloudFront 400 error when adding an `errorPage` this way:

```diff
--- sst.config.ts
+++ sst.config.ts
@@ -21,6 +21,7 @@ export default $config({
 
     const site = new sst.aws.StaticSite("Extensions", {
       path: "repo",
+      errorPage: "404.html",
       domain: extDomains[$app.stage],
     });
```

**Comment by mlafeldt** [comment]
I just ran into this bug. Here's a quick PR to fix it: #635
